frontend/src/store/bucketStore.ts

```diff
@@ -105,12 +111,12 @@ export const useBucketStore = defineStore('bucket', () => {
   async function getBucketPermissionsForUser(bucketId: string) {
     try {
       loading.value = true;
-  
+
       if (currentUser.value) {
-        const bucketPerms = 
+        const bucketPerms =
           (await permissionService.bucketSearchPermissions({ bucketId, userId: currentUser.value.userId })).data;
-        if(bucketPerms && bucketPerms[0] && bucketPerms[0].permissions) {
-          selectedBucketPermissionsForUser.value =bucketPerms[0].permissions;
+        if (bucketPerms && bucketPerms[0] && bucketPerms[0].permissions) {
```

Comment:
the permission search in coms wll return an empty array if no perms were found.
maybe you can skip line 115-118 and do:
`selectedBucketPermissionsForUser.value = await permissionService.bucketSearchPermissions({ bucketId, userId: currentUser.value.userId })).data.[0].permissions`